### DISCUSSION TITLE
How to cross reference an Appendix

### DISCUSSION THREAD

**Discussion by jpmorr** [quarto-dev/quarto-cli#1820] [Q&A] [answered]
I'm trying to add a link to a specific appendix into a chapter but it keeps showing up as `Chapter A` instead of `Appendix A`.

I'm using `# Appendix A {#sec-AppendixA}` as the section title in the appendix file and then calling `@sec-AppendixA` in the chapter in the requires location but it always seems to just return chapter.

Is there a specific appendix prefix instead of `sec` that I should be using? I didn't see any specific example about referencing an appendix chapter.

**Comment by jjallaire** [comment, marked as answer]
This should be fixed in our pre-release: https://quarto.org/docs/download/prerelease.html

**Reply by jpmorr** [reply]
Yes, it is fixed in this version. Thanks,

**Reply by AustenLamacraft** [reply]
I can't seem to reference sections from another file, but it sounds like it's working for you. Is there some particular setting required?

**Reply by dragonstyle** [reply]
This should be supported for books without doing anything special. For non-book projects (e.g. websites), this doesn't work and you can simple link to `qmd` files and we'll automatically resolve those links to the correct output when the website is rendered.
